### ISSUE TITLE
CI Testing Environments

### ISSUE THREAD

**Issue by sonowz** [sonowz/graduate-adventure#35]
README.md says
```
Recommended Requirements

Python ≥ 3.5
node ≥ 6.x
```
But .travis.yml says
```
python:
  - "3.5"
  - "2.7"
  - "pypy"
```
Since pythons are used only in server-side, I think we don't have to test `python 2.7.`
What's your opinions?

**Comment by dj-shin** [comment]
I also think we can reduce the number of test environments. The reason why I left those settings on CI is just it would be good if this project can run on multiple environments, but it is just an option, not a necessity.

**Comment by dj-shin** [comment]
Resolved with #51
